src/app/pages/wallets/my-wallets/modals/dialog-create-or-update-wallet/wallet-form-modal.component.html

```diff
@@ -0,0 +1,11 @@
+<form [formGroup]='form'>
```

Comment:
Instead of `my-wallets/modals/dialog-create-or-update-wallet` lets place it in `my-wallets/wallet-form-modal`, that should be enough :) And directory will match in that case also component files names.